modulemd/v2/tests/ModulemdTests/translation_helpers.py

```diff
@@ -68,6 +68,52 @@ def test_translation_catalog_from_index(self):
         # There are 34 unique summaries and descriptions in f29.yaml
         self.assertEquals(len(catalog), 34)
 
+    def test_translations_from_catalog(self):
+        translation_files = ["../test_data/nl.po"]
+
+        catalogs = dict()
+        for f in translation_files:
+            with open(f, 'r') as infile:
+                catalog = pofile.read_po(infile)
+                self.assertTrue(catalog)
+
+                # Check if the catalog is internally consistent
+                _, err = catalog.check()
+                self.assertNone(err)
+                catalogs[catalog.locale_identifier] = catalog
+
+        # Create a Modulemd.ModuleIndex object containing the YAML stream
+        index = Modulemd.ModuleIndex.new()
+        yaml_path = "../test_data/f29.yaml"
+        ret, failures = index.update_from_file(yaml_path, True)
+        self.assertIsNotNone(index)
+
+        err = get_modulemd_translations_from_catalog(catalogs, index)
+        self.assertNone(err)
+
+        for locale, catalog in catalogs.items():
+            for msg in catalog:
+                locations = [x[0] for x in msg.locations]
+                for location in locations:
+                    props = location.split(";")
```

Comment:
Please make this a utility function that returns named attributes instead of referencing it with array index values.

So something like `(module_name, stream_name, location_type, profile_name) = split_location(location)`

Reply:
`profile_name` should return `None` for `location_type` of "summary" or "description".